concent_api/conductor/tasks.py

```diff
@@ -79,12 +92,21 @@ def upload_acknowledged(
     result_file_size: str,
     result_package_hash: str,
 ):
+    log_string_message(
+        logger,
+        f'Upload acknowledgment starts. SUBTASK_ID: {subtask_id} ',
+        f'Source_file_size {source_file_size} ',
+        f'Source_package_hash: {source_package_hash} ',
+        f'Result_file_size: {result_file_size} ',
+        f'Result_package_hash: {result_package_hash} '
+    )
     assert isinstance(subtask_id, str)
 
     try:
         verification_request = VerificationRequest.objects.get(subtask_id=subtask_id)
     except VerificationRequest.DoesNotExist:
-        logging.error(
```

Comment:
Changed level of logging- i think it should not be error. If it should I will change it

Reply:
It should be an error.

This situation means that the control cluster is acknowledging an upload for a verification request that does not exist. It should never happen because Conductor is not supposed to send `upload_finished` if it does not exist. It means that something somewhere is seriously wrong.

Reply:
Done